### DISCUSSION TITLE
Local or Cloud integration?

### DISCUSSION THREAD

**Discussion by igd001** [petretiandrea/home-assistant-tapo-p100#398] [General]
it is not clear to me if this integration need internet or work locally. I understood it needs authentication through the cloud, but how often does it happen?. If internet stop will my automations stop as well or will continue working?. Thanks for clarifying!!

**Comment by petretiandrea** [comment]
Is a local integration but also require the device to be connected to cloud in order to synchronize it with password changes and so on

**Comment by igd001** [comment]
Thanks!!!IgnacioEl 21 jun 2023, a las 22:23, petretiandrea ***@***.***> escribió:﻿
Is a local integration but also require the device to be connected to cloud in order to synchronize it with password changes and so on
